test/parallel_testsuite.py

```diff
@@ -159,7 +168,19 @@ def combine_results(self, result, buffered_results):
       os.set_blocking(sys.stderr.fileno(), True)
 
     for r in results:
+      # Merge information of flaky tests into the test result
+      if r.test_result == 'success' and r.test_short_name() in flaky_tests:
+        r.test_result = 'warnings'
```

Comment:
Is this "warnings" test result only surfaced via the visualization, or does it show up on the command line output too?

Reply:
It only shows up in visualization. Python unit test runner does not have a concept of test finishing with warnings to my knowledge.